requirements.txt

```diff
@@ -6,6 +6,6 @@ networkx
 git+https://github.com/lsst/resources@main#egg=lsst-resources
 git+https://github.com/lsst/daf_butler@main#egg=lsst-daf-butler
 git+https://github.com/lsst/utils@main#egg=lsst-utils
-git+https://github.com/lsst/pipe_base@main#egg=lsst-pipe-base
+git+https://github.com/lsst/pipe_base@tickets/DM-41606#egg=lsst-pipe-base
```

Comment:
Also here don't forget to change it back when merging.

Reply:
There is a GitHub action that prevents merging unless this has been removed.